cmd/state/commands/erigon2.go

```diff
@@ -226,17 +243,24 @@ func Erigon2(genesis *core.Genesis, chainConfig *params.ChainConfig, logger log.
 			}
 			return h
 		}
-		txNum++ // Pre-block transaction
-		if txNum, _, err = runBlock2(trace, txNum, readWrapper, writeWrapper, chainConfig, getHeader, b, vmConfig); err != nil {
-			return fmt.Errorf("block %d: %w", blockNum, err)
+
+		if engine.Type() != params.CliqueConsensus {
+			txNum++ // Pre-block transaction
```

Comment:
Hmm, I thought that Clique would also have pre-block and post-block transactions. Maybe you have not migrated your DB?

Reply:
Checked current migration via `integration print_migration` -- v5, tried to apply `run_migration`, but no space left, so copying data to external drive and check again.

Hope you are right and Clique also has system pre\post block txns. 
I thought migrations is not the case since I pulled Goerli from scratch yesterday.

Reply:
@awskii there is 1 more migration in the list in `devel`: https://github.com/ledgerwatch/erigon/blob/bb711676a4d8523e6e45f0ce7887010781304bf2/migrations/migrations.go#L35